packages/net/tcp_connection.pony

```diff
@@ -197,6 +197,9 @@ actor TCPConnection
   var _shutdown: Bool = false
   var _shutdown_peer: Bool = false
   var _in_sent: Bool = false
+  // _pending is used to avoid GC prematurely reaping memory.
+  // See GitHub bug 2526 for more.  It looks like a write-only
+  // data structure, but its use is important until the Pony runtime changes.
```

Comment:
"until the Pony runtime changes"?

That part is leading and ambiguous to me. I would suggest either expanding the statement to describe the nature of the change, or if there is no specific change in mind, just say something like "due to the way the Pony runtime works".